Formula/s/serpl.rb

```diff
@@ -0,0 +1,18 @@
+class Serpl < Formula
+  desc "Simple terminal UI for search and replace"
+  homepage "https://github.com/yassinebridi/serpl"
+  url "https://github.com/yassinebridi/serpl/archive/refs/tags/0.3.1.tar.gz"
+  sha256 "fa0d5b9de9a8820ba46d8d2a7e4a20fc99bcc433816e9368fe547bf7b6c1d1b2"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "ripgrep"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/serpl --version")
```

Comment:
```suggestion
    assert_match version.to_s, shell_output(bin/"serpl --version")
    
    assert_match "a value is required for '--project-root <PATH>' but none was supplied",
    	shell_output(bin/"serpl --project-root 2>&1", 1)
```

Reply:
Thanks for the assistance!

Reply:
The `shell_output("bin/string with a space in it")` didn't trip RuboCop in the first place because of https://github.com/Homebrew/brew/pull/17856, so I'm gonna switch it back.